djmoney/contrib/exchange/backends/fixer.py

```diff
@@ -15,4 +15,4 @@ def __init__(self, url=settings.FIXER_URL, access_key=settings.FIXER_ACCESS_KEY)
         self.access_key = access_key
 
     def get_params(self):
-        return {"access_key": self.access_key}
+        return {"apikey": self.access_key}
```

Comment:
Why is this change added? Isn't the fixer api expecting an `access_key` parameter?
https://fixer.io/documentation

Reply:
Thanks @spaut33 for the quick response. It makes sense, but isn't the `apikey` suppose to be included in the headers in that case?
Sorry for the questions but a code I'm working on started failing after upgrading djmoney to 3.1.0 and I'm trying to figure out what's the issue.

Reply:
It works both ways, as a request parameter and as a header. However, I believe that APILayer may not be following the correct convention, as custom fields in headers should start with the "X-" prefix.

Reply:
Same here. Fixed by creating API key through ApiLayer and adding `FIXER_URL = "https://api.apilayer.com/fixer/latest"` to my `settings.py`

Reply:
But `apilayer` seems unreliable — getting timeouts accessing the API. Could we maybe send `access_key` and `apikey` params to suit both APIs?

Reply:
@ignat14 @fedosov Thank you for your comments. I updated the [issue](https://github.com/django-money/django-money/issues/675)